GeosCore/seasalt_mod.F90

```diff
@@ -418,11 +419,11 @@ SUBROUTINE ChemSeaSalt( Input_Opt,  State_Chm, State_Diag,                 &
     !----------------------------------------
     ! Sea salt emissions for extra APM bins
     !----------------------------------------
-    SEASALTS = &
-     State_Chm%Species(:,:,:,APMIDS%id_SEABIN1:(APMIDS%id_SEABIN1+NSALTBIN-1))
-    CALL SRCSALTBIN( SEASALTS, State_Grid, State_Met )
-    State_Chm%Species(:,:,:,APMIDS%id_SEABIN1:(APMIDS%id_SEABIN1+NSALTBIN-1)) &
-      = SEASALTS
+
+    DO N = 1, NSALTBIN
```

Comment:
Is State_Chm%Species(N)%Conc now referenced in the SRCSALTBIN routine?

Reply:
Yes, the species IDs are passed to SRCSALTBIN and used to retrieve concentrations from State_Chm%Species. See [here](https://github.com/geoschem/geos-chem/blob/1085010733aa9fc0276d3adfdb918970895b788a/GeosCore/seasalt_mod.F90#L1670-L1672).